### DISCUSSION TITLE
PartialVisible for last slide

### DISCUSSION THREAD

**Discussion by sanderkooger** [nolimits4web/swiper#4807] [Q&A]
### Clear and concise description of the problem

We are workign on an interface for a video streaming website. in this interfce we use sliders with multiple items ( responsive (phone 3, tablet 5, lg screen 7) we would liek to display part of the slide that comes next to trigger users to know they can swipe an scroll. 

### Suggested solution

implement a partialVissible prop that takes a number that tells how much of the next slide should be visible. 

![image](https://user-images.githubusercontent.com/19397354/128044444-878faada-be98-410a-9e28-70192824a21a.png)

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [X] I'm willing to open a PR

**Comment by sanderkooger** [comment]
@onair-lena would you be willing to contribute?

**Reply by onair-lena** [reply]
@nolimits4web Vadimir, do you know if its possible to have a non centered view with a partially visible last slide? 

Im also looking onto disabling vertical scrolling, we have a page filled with sliders that are stacked on top of each other, but if i enable mousewheel, scrolling down the page stops working. 

We tried ForceToAxis, but it did not resolve the behaviour.

**Reply by nolimits4web** [reply]
`slidesPerView: 'auto'` is what you are looking for https://swiperjs.com/demos/120-slides-per-view-auto/core.html

**Reply by onair-lena** [reply]
Hey @nolimits4web, Thank you for the quick answer!

We just tried the proposed solution, but it does not allow us to show more than 1.3 images. 

![image](https://user-images.githubusercontent.com/47395779/128054534-f1c0078b-1754-4f35-be81-c916c9f6757f.png)


in our usecase we are looking to display 3.2 images for mobile, 5.2 for tablet and 7.2 for desktop. 

Any suggestions?

**Reply by nolimits4web** [reply]
Look at the demo, you need to set slide's width in CSS according to your needs, `3.2` is `width: calc(100% / 3.2)`, etc.

**Reply by onair-lena** [reply]
This solves issue. Thanks for help!
